spring-cloud-gcp-logging/src/main/java/com/google/cloud/spring/logging/StackdriverJsonLayout.java

```diff
@@ -157,6 +162,20 @@ public void setCustomJson(String json) {
 		this.customJson = gson.fromJson(json, Map.class);
 	}
 
+	/**
+	 * Add additional logging enhancers that implement {@link JsonLoggingEventEnhancer}.
+	 * @param enhancerClassName class name of the layout enhancer
+	 */
+	public void addLoggingEventEnhancer(String enhancerClassName) {
+		try {
+			Class<JsonLoggingEventEnhancer> clz =
+					(Class<JsonLoggingEventEnhancer>) Loader.loadClass(enhancerClassName.trim());
+			loggingEventEnhancers.add(clz.getDeclaredConstructor().newInstance());
+		}
+		catch (Exception ex) {
```

Comment:
Is this for `ClassNotFoundException`?

Reply:
Yeah there's a couple checked exceptions -- ClassNotFound, MethodNotFound (for invoking the constructor), and a few others.